Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+#!groovy
```

Comment:
I have considered declarative pipelines. My understanding is that when you are doing things like dynamically creating stages (in this case matrixing between goversions & archs) scripted  is preferred to declarative.